docs/getting-started/siem-ui.asciidoc

```diff
@@ -161,7 +161,7 @@ There are also tabs for viewing and investigating specific types of data:
 {apm-app-ref}/apm-getting-started.html[Elastic APM] are monitored by default)
 * TLS: handshake details
 * Anomalies: anomalies discovered by <<machine-learning, machine learning jobs>>
-* <<det-engine-terminology, External alerts>>: alerts received from
+* External alerts: alerts received from
 external monitoring tools, such as Elastic Endpoint Security
```

Comment:
@joepeeples I'm checking the terminology of `Elastic Endpoint Security` here. AFAIK, we're not using this term anymore except for when referring to the endpoint rule -- but double checking my sanity.

Reply:
@jmikell821 Correct, "Elastic Endpoint Security" is now "Elastic Defend." It's a bit moot in this case, though, because this file `siem-ui.asciidoc` is in the old (retired?) [SIEM Guide](https://www.elastic.co/guide/en/siem/guide/current/siem-ui-overview.html) book; I don't think this content will ever show up in current Elastic Security docs. That's why it wasn't revised when the integration was renamed, or re-renamed.

For the sake of this PR, no further changes needed; we can decide separately if we need to follow up on the terminology.

Reply:
@joepeeples true. Customers won't see it unless they're looking at `7.8` and earlier -- so I'm not too worried about it. Thanks!

Reply:
The "Elastic Endpoint Security" terminology was also used in the "endpoint exception" term that I carried over into the glossary: https://www.elastic.co/guide/en/elastic-stack-glossary/current/terms.html#e-glos. If that's not a useful term anymore, let me know and we can correct or remove that entry.

Reply:
@lcawl Actually that particular instance is OK as is, too -- another case of "It Depends"! 🙃 The "Elastic Endpoint Security rule" mentioned in that glossary entry has not yet been renamed to match the integration's name changes, so it's another rare case where we don't need to change it in docs (yet?). If they ever do rename the detection rule, we can update the glossary at that time.